code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -117,6 +117,30 @@
 						return
 				//end vampire codes
 
+				if(src.species.name == "Wryn")

```

Comment:
This all needs moved to an attack_hand handler on the species itself, having this stuff on a block of core attack hand code is very bad